Example/BitmovinConvivaAnalytics/Base.lproj/Main.storyboard

```diff
@@ -91,6 +91,17 @@
                                     <action selector="setupPlayer:" destination="vXZ-lx-hvc" eventType="touchUpInside" id="pdM-M2-Cgh"/>
                                 </connections>
                             </button>
+                            <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="YLt-uJ-a1f">
+                                <rect key="frame" x="20" y="557.33333333333337" width="390" height="35"/>
+                                <state key="normal" title="Send custom event">
+                                    <color key="titleColor" red="0.1215686275" green="0.6705882353" blue="0.8862745098" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                </state>
+                                <buttonConfiguration key="configuration" style="filled" title="Start session"/>
+                                <connections>
+                                    <action selector="sendCustomEvent:" destination="vXZ-lx-hvc" eventType="touchUpInside" id="y4i-Ad-xIh"/>
+                                    <action selector="startSession:" destination="vXZ-lx-hvc" eventType="touchUpInside" id="vOM-BI-JTG"/>
+                                </connections>
+                            </button>
                         </subviews>
```

Comment:
There is a new `Start session` button which can be used to test this manually in the example. See PR description for manually testing instructions.